### PR TITLE
fixes credential issue

### LINKED ISSUE #317: database connection
## Problem
While connectiing to the database wveryone running the project locally need to change username and password and accidently most of the contributors push code with their credentials .

## Solution

We can have a sample.env which include username and password field and every time when user runs the projects locally will create .env files and copy the data from sample.env and respective value of username and password , hence while pushing changes credentials remains safe 

Will add respective instructions in Documentation.

### PR DESCRIPTION
**Pull request by Sukriti-sood**
Fixes #317 

Updated  [Document](https://www.notion.so/Project-setup-Public-1a647ed8515c485f99f38e717acfa61b)

### CODE REVIEW

File: db-connect.php [hunk 1]
```diff
@@ -1,10 +1,20 @@
 <?php
```
Comment (siv2r) [inline db-connect.php:1]: Is there any reason that `db-connect.php` was moved out to the `templates` folder?

**Review by siv2r — CHANGES_REQUESTED** [review]

File: db-connect.php (hunk 1 above)
Comment (Sukriti-sood) [reply db-connect.php:1]: Yes , I tried using the composer inside the template folded so , I moved out the file .

Comment (siv2r) [reply db-connect.php:1]: Did you test the following pages after this?
- login/signup
- data -> summary
- data -> overview
- search (enter some pair id)
- global match
- pairwise match

Comment (siv2r) [reply db-connect.php:1]: Since the functionality of most of the pages depends on this file.

Comment (Sukriti-sood) [reply db-connect.php:1]: Ya I tested all of them

Comment (siv2r) [reply db-connect.php:1]: Awesome!

**Pull request merged**